### ISSUE TITLE
Issue when importing with some custom fields.

### ISSUE THREAD

**Issue by DylanDKnight** [snipe/snipe-it#5802]
#### Please confirm you have done the following before posting your bug report:

- [X] I have enabled debug mode 
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

**Describe the bug**
When importing some custom fields, others work fine, some don't. The ones that work have the same type of field as the others, and there is no weird encoding going on by the looks of it, I've removed all other rows from the CSV that don't contain data, and the fieldset is correctly applied to the asset type as the other custom fields work that are in the same fieldset.

**To Reproduce**
1. Go to Import, import CSV.
2. Process and map CSV to custom fields in SnipeIT
3. Fields don't populate and are still Null.
4. No Error.

**Expected behavior**
The expected behavior is the custom fields are populated.

**Screenshots**
Custom fields on the asset page, these are in the "PC Specs" Field set
![image](https://user-images.githubusercontent.com/17187287/42497799-5d75146a-8422-11e8-8e8e-a9b6be16c2f8.png)

When updating the fields manually they go in fine.
![image](https://user-images.githubusercontent.com/17187287/42497861-974b481c-8422-11e8-9e16-5daf1a858c51.png)

Some custom fields do work with the upload, e.g. Storage, RAM. Purchase cost and Date.

**Server (please complete the following information):**
 - Snipe-IT Version: v4.5.0 build 3701
 - OS: [Ubuntu 16.04]
 - Web Server: [nginx/1.10.3 (Ubuntu)]
 - PHP Version 7.0.30-0ubuntu0.16.04.1

**Desktop (please complete the following information*
 - OS: [Windows 10 1709 | Build 16299.492]
 - Browser [Chrome]
 - Version 67.0.3396.99

**Error Messages**
- No Error in Debug Console
- No Stackstace
- No Error message in console
- Unable to confirm on DEMO as Import is disabled.
- No Errors in NGINX Log

**Additional context**
- Recently fresh install, followed by an upgrade to the latest version.
- Ubuntu 16.04 and a Xenserver VM
- Git + install.sh
- No errors occurred when installing
- No data manually edited in DB
- No errors reported in the MySQL logs.

I have deleted all columns at the bottom and side of the CSV uploaded, opened in Notepad++ and Atom, no visible encoding or special character issues.

**Comment by EarlRamirez** [comment]
@DylanDKnight, when you try to import from the Web UI, do see these custom fields in the drop-down menu?

**Comment by DylanDKnight** [comment]
Yeah, they are mappable when importing via drop down.

The models also had the correct fieldsets, set to them.

I am importing a spreadsheet with 24 columns of data.

I have noticed when importing a spreadsheet with only say 15 rows all the
data imports, but when importing with say 100 rows, a lot of columns are
missing. So that could be a starting point to replicate?

I got around it by writing something in Python to do it via the API.

But from all the logs I looked at there was a single error message, it's
almost like the data wasn't even there to go into the DB.
